contracts/stellar-interchain-token-service/src/tests/deploy_remote_interchain_token.rs

```diff
@@ -77,7 +80,7 @@ fn deploy_remote_interchain_token_succeeds() {
             its_hub_address,
             payload,
             &sender,
-            gas_token.clone(),
+            _gas_token.clone(),
```

Comment:
```suggestion
            Some(gas_token.clone()),
```
After fixing the confusing setup API

Reply:
updated